src/app/components/card-item/card-item.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, Input,ViewChild } from '@angular/core';
+import { Task } from 'src/app/models/Task.model';
+import { formatDate } from '@angular/common';
+import { TaskService } from 'src/app/services/task.service';
+import { UtilityService } from 'src/app/services/utility.service';
+
+@Component({
+  selector: 'app-card-item',
+  templateUrl: './card-item.component.html',
+  styleUrls: ['./card-item.component.css'],
+})
+
+export class CardItemComponent {
+
+  constructor(private taskService: TaskService, private utilityService: UtilityService){}
+  
+  @Input() taskLists: Task[] = [];
+  TODAY: Date = new Date();
+  formattedDate = formatDate(this.TODAY, 'dd.MM.yy', 'en-GB');
+  taskList = this.taskService.gettaskList();
+  utility = this.utilityService;
+}
```

Comment:
EOF newline

Reply:
configured eslintrc.json for eol (end of line)